packages/react-dom/src/events/__tests__/BeforeInputEventPlugin-test.internal.js

```diff
@@ -156,32 +161,17 @@ describe('BeforeInputEventPlugin', function() {
   // keyUp.
   const Expected_Webkit = () => [
     {type: ModuleCache.SyntheticCompositionEvent, data: {}},
-    {type: null},
-    {type: null},
     {type: ModuleCache.SyntheticInputEvent, data: {data: 'A'}},
-    {type: null},
-    {type: null}, // textinput of A
-    {type: null},
-    {type: null}, // keyUp of 65
-    {type: null},
+    // textinput of A
+    // keyUp of 65
     {type: ModuleCache.SyntheticInputEvent, data: {data: 'abc'}},
-    {type: null},
-    {type: null}, // textinput of abc
-    {type: null},
-    {type: null}, // keyUp of 32
-    {type: null},
+    // textinput of abc
+    // keyUp of 32
     {type: ModuleCache.SyntheticInputEvent, data: {data: 'xyz'}},
-    {type: null},
-    {type: null}, // textinput of xyz
-    {type: null},
-    {type: null}, // keyUp of 32
+    // textinput of xyz
+    // keyUp of 32
     {type: ModuleCache.SyntheticCompositionEvent, data: {data: 'Hello'}},
-    {type: null},
-
     // Emoji test
-    {type: null},
-    {type: null},
-    {type: null},
```

Comment:
As far as I can tell, `type: null` are all of the `null` values from `extractEvents`, which could be a permutation of:

```
[null, null] // (neither a composition or beforeinput event)
[composition, beforeInput]
[null, beforeInput]
[composition, null]
```

So my change makes this change one of:

```
null
composition
beforeinput
[composition, beforeinput]
```

`null` gets filtered out by the EventPluginHub.extractEvents. It just does a basic null check, so `[null, null]` gets through.